src/main/java/io/confluent/connect/hdfs/HdfsSinkConnectorConfig.java

```diff
@@ -118,6 +118,21 @@
       + "provide fixed width filenames that can be ordered by simple lexicographic sorting.";
   public static final int FILENAME_OFFSET_ZERO_PAD_WIDTH_DEFAULT = 10;
 
+  public static final String HDFS_AUTHENTICATE_KERBEROS_CONFIG = "hadoop.authentication.kerberos";
```

Comment:
Seems inconsistent to use the prefix `hadoop.` here but `hdfs.` prefix elsewhere. Is this intentional (especially since the variable name and config key don't match)?